corehq/apps/sms/templates/sms/chat.html

```diff
@@ -25,6 +25,7 @@
         <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
+        <script src="{% static 'style/js/hq.helpers.js' %}"></script>
```

Comment:
Feel free to close this if you're concerned about it. I'd rather be ideological about standards, but we don't currently have the test coverage to guarantee that scenario won't happen.